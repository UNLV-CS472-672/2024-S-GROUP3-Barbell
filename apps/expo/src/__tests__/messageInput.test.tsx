import { render, screen } from '@testing-library/react-native'

import MessageInput from '~/components/message/messageInput'

test('MessageInput', async () => {
  render(<MessageInput />)
  expect(screen.toJSON()).toMatchSnapshot()
})
