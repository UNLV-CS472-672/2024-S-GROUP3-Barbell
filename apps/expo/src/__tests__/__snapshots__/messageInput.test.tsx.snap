// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageInput 1`] = `
<View
  onLayout={[Function]}
  style={
    {
      "flex": 1,
      "paddingBottom": 10,
    }
  }
>
  <View
    style={
      {
        "flex": 1,
        "justifyContent": "flex-end",
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "flexDirection": "row",
          "paddingHorizontal": 10,
        }
      }
    >
      <TextInput
        keyboardAppearance="dark"
        placeholder="Message..."
        placeholderTextColor="#CACACA"
        style={
          {
            "borderColor": "#272727",
            "borderRadius": 20,
            "borderWidth": 1,
            "color": "white",
            "flex": 1,
            "padding": 10,
          }
        }
      />
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "backgroundColor": "#48476D",
              "borderRadius": 20,
              "marginLeft": 10,
              "padding": 10,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
  </View>
</View>
`;
